src/test/java/edu/cornell/kfs/fp/document/service/impl/CuDisbursementVoucherDefaultDueDateServiceImplTest.java

```diff
@@ -1,128 +1,110 @@
 package edu.cornell.kfs.fp.document.service.impl;
 
-import static org.junit.Assert.*;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
 
 import java.sql.Date;
-import java.text.SimpleDateFormat;
-import java.util.Calendar;
-import java.util.Locale;
+import java.time.LocalDate;
+import java.time.Month;
 
 import org.apache.log4j.Level;
 import org.apache.log4j.Logger;
-import org.junit.After;
-import org.junit.Before;
-import org.junit.Test;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.kuali.kfs.core.api.datetime.DateTimeService;
 import org.kuali.kfs.coreservice.framework.parameter.ParameterService;
 import org.kuali.kfs.fp.document.DisbursementVoucherDocument;
-import org.kuali.kfs.core.api.datetime.DateTimeService;
 import org.mockito.Mockito;
 
 import edu.cornell.kfs.fp.CuFPParameterConstants;
-import edu.cornell.kfs.sys.CUKFSConstants;
+import edu.cornell.kfs.sys.service.impl.TestDateTimeServiceImpl;
 
 public class CuDisbursementVoucherDefaultDueDateServiceImplTest {
     private CuDisbursementVoucherDefaultDueDateServiceImpl cuDisbursementVoucherDefaultDueDateServiceImpl;
-    private SimpleDateFormat dateFormat;
 
-    @Before
+    @BeforeEach
     public void setUp() throws Exception {
         Logger.getLogger(CuDisbursementVoucherDefaultDueDateServiceImpl.class.getName()).setLevel(Level.DEBUG);
         cuDisbursementVoucherDefaultDueDateServiceImpl = new CuDisbursementVoucherDefaultDueDateServiceImpl();
-        dateFormat = new SimpleDateFormat(CUKFSConstants.DATE_FORMAT_yyyyMMdd, Locale.US);
     }
 
-    @After
+    @AfterEach
     public void tearDown() throws Exception {
         cuDisbursementVoucherDefaultDueDateServiceImpl = null;
-        dateFormat = null;
     }
 
     @Test
-    public void testFindDefaultDueDateToday() {
-        prepCuDisbursementVoucherDefaultDueDateServiceImpl("1", Calendar.getInstance());
-        Calendar expectedCalendar = Calendar.getInstance();
-        expectedCalendar.add(Calendar.DAY_OF_MONTH, 1);
+    public void testFindDefaultDueDateToday() throws Exception {
+        prepCuDisbursementVoucherDefaultDueDateServiceImpl("1", LocalDate.of(LocalDate.now().getYear(), LocalDate.now().getMonth(), LocalDate.now().getDayOfMonth()));
```

Comment:
Could this spot just use `LocalDate.now()` directly, rather than picking apart the current year/month/day and manually adding them to a new instance?